setup.cfg

```diff
@@ -1,3 +1,6 @@
 [bdist_wheel]
 universal = 1
 
+[zest.releaser]
+create-wheel = yes
```

Comment:
We might want to add ``version-levels = 3`` here, for semver-style versioning.

Reply:
Do we really need it ? The default value of 0 should provide the same result.